Gruvo/Gruvo/Controllers/TokenManager.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.IdentityModel.Tokens.Jwt;
+using Microsoft.IdentityModel.Tokens;
+using System.Security.Claims;
+using System.Text;
+using Microsoft.Extensions.Configuration;
+using Gruvo.DAL;
+using Gruvo.Controllers;
+
+namespace Gruvo.Models
```

Comment:
It should not be in "Controllers" directory. Let it be in BLL (business logic layer) folder